uint8 gimbal_mode
uint8 chassis_mode